heron/statemgrs/src/java/com/twitter/heron/statemgr/zookeeper/curator/CuratorStateManager.java

```diff
@@ -310,4 +320,19 @@ public void processResult(CuratorFramework aClient, CuratorEvent event) throws E
       return deleteNode(getSchedulerLocationPath(topologyName));
     }
   }
+
```

Comment:
What we have here is only intended for developers while troubleshooting, not end users per se, so I don't see a need at this point to create a distributable tool. End users would have a higher level abstraction in the heron-explorer. 

The command to run it currently is already pretty simple:

local:
`bazel run heron/statemgrs/src/java:localfs-statemgr-unshaded -- ExclamationTopology`
zk:
`bazel run heron/statemgrs/src/java:zookeeper-statemgr-unshaded -- ExclamationTopology my.zookeeper.host.com` 

We could make a wrapper bash script I suppose. I don't see the need for a python version, since it would just do the same thing as the java.